pkg/splunk/enterprise/configuration.go

```diff
@@ -315,6 +308,76 @@ func addSplunkVolumeToTemplate(podTemplateSpec *corev1.PodTemplateSpec, name str
 	}
 }
 
+// addPVCVolumes adds pvc volumes to statefulSet
+func addPVCVolumes(cr splcommon.MetaObject, spec *enterprisev1.CommonSplunkSpec, statefulSet *appsv1.StatefulSet, labels map[string]string, volumeType string) error {
+	// prepare and append persistent volume claims if storage is not ephemeral
```

Comment:
What are the usecases where we may need to 'append' multiple PersistentVolumeClaims to the same pods? What if a user is attempting to replace one PersistentVolume with another?